### PR TITLE
Feature/feature scaffolding

### PR DESCRIPTION
**Pull request by lennynilsson**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ const querystring = require('querystring')
 const HockeyApp = require('hockeyapp-api-wrapper')
 const yaml = require('js-yaml')
 const cucumberHtmlReporter = require('cucumber-html-reporter')
+const Mustache = require('mustache')
```
Comment (eridem) [inline index.js]: - Use Mustache just to attach a tag on the top is a bit excessive :) Please remove and adapt.
- The library `changeCase` is not needed. There is a function that formats already any name in the library `validation.formatName`
- When you use `validation.formatName`, inject the library `latinize` on there.

Comment (eridem) [inline index.js]: Merge into `develop`.

Comment (lennynilsson) [inline index.js]: The npm package 'change-case' provides snake case also, which cannot be done with validation.formatName. Another option is to replace validation.formatName with the npm packages 'latinize' and 'param-case', and then also include the npm package 'snake-case' for the tag. It is a nice feature to use 'sentence-case' just to make it easy to throw whatever string as a feature name.

In conclusion; I used 'change-case' in order to cover to be able to throw whatever name and end up with a nice formatted output.

I will change for the branch develop

Comment (lennynilsson) [inline index.js]: @eridem I have updated the pull request now.

**Pull request closed without merging**